docs/site/pages/getting-started.mdx

```diff
@@ -198,3 +198,44 @@ class SimplePlayerFragment(override val flow: String) : PlayerFragment() {
     }
 }
 ```
+
+### Release Optimization
+
+The Android Player dependency automatically pulls in both the `dev` as well as the `prod` versions of the underlying JS scripts.
+
+The `dev` scripts are handy for debugging shared JS code through the JS debugger in debug builds, but they are not necessary for consumer facing release builds.1
```

Comment:
```suggestion
The `dev` scripts are handy for debugging shared JS code through the JS debugger in debug builds, but they are not necessary for consumer facing release builds.
```

Reply:
hmmm my keyboard's enter key was adding these all over the place, i think this is the 4th one